docs/developer_guide/getting_started.md

```diff
@@ -89,26 +87,26 @@ await mainFunction();
 
 async function mainFunction() {
   // create a random keys-only (offline) stagenet wallet
-  var walletKeys = await MoneroWalletKeys.createWallet({networkType: MoneroNetworkType.STAGENET, language: "English"});
+  let walletKeys = await MoneroWalletKeys.createWallet({networkType: MoneroNetworkType.STAGENET, language: "English"});
 
-  console.log("Seed phrase: " + await(walletKeys.getMnemonic()));
-  console.log("Address: " + await(walletKeys.getAddress(0,0))); // MoneroWallet.getAddress(accountIndex, subAddress)
+  console.log("Mnemonic seed phrase: " + await(walletKeys.getMnemonic()));
```

Comment:
`await(walletKeys.getMnemonic())` should be `await walletKeys.getMnemonic()`, same for other instances.